google/cloud/resourcesettings/quickstart/README.md

```diff
@@ -157,7 +157,7 @@ set GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=%cd%\roots.pem
 ```
 
 [bazel-install]: https://docs.bazel.build/versions/main/install.html
-[quickstart-link]: https://cloud.google.com/resourcesettings/docs/quickstart
+[quickstart-link]: https://cloud.google.com/resource-manager/docs/quickstarts
```

Comment:
hmm, the doc structure is weird. maybe we should link to this: https://cloud.google.com/resource-manager/docs/resource-settings/manage-resource-settings